SemanticReleaseError: Not running from a git repository.

**Describe the bug**

When running semantic release, I receive the following error:


```

[9:11:51 PM] [semantic-release] › ℹ  Running semantic-release version 15.13.28
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/npm"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/npm"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/npm"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
[9:11:52 PM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
[9:11:52 PM] [semantic-release] › ✔  Run automated release from branch master
[9:11:52 PM] [semantic-release] › ℹ  Start step "fail" of plugin "@semantic-release/github"
[9:11:52 PM] [semantic-release] [@semantic-release/github] › ℹ  Verify GitHub authentication
[9:11:52 PM] [semantic-release] › ✖  Failed step "fail" of plugin "@semantic-release/github"
[9:11:52 PM] [semantic-release] › ✖  EINVALIDGITHUBURL The git repository URL is not a valid GitHub URL.
The semantic-release repositoryUrl option must a valid GitHub URL with the format <GitHub_or_GHE_URL>/<owner>/<repo>.git.

By default the repositoryUrl option is retrieved from the repository property of your package.json or the git origin url (https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) of the repository cloned by your CI environment.

[9:11:52 PM] [semantic-release] › ✖  ENOGITREPO Not running from a git repository.
The semantic-release command must be executed from a Git repository.

The current working directory is /home/runner/work/stripe-elements/stripe-elements.

Please verify your CI configuration to make sure the semantic-release command is executed from the root of the cloned repository.

##[error]AggregateError: 
    SemanticReleaseError: Not running from a git repository.
        at module.exports (/home/runner/work/_actions/cycjimmy/semantic-release-action/v2/node_modules/semantic-release/lib/get-error.js:6:10)
        at module.exports (/home/runner/work/_actions/cycjimmy/semantic-release-action/v2/node_modules/semantic-release/lib/verify.js:10:17)
        at async run (/home/runner/work/_actions/cycjimmy/semantic-release-action/v2/node_modules/semantic-release/index.js:60:3)
        at async module.exports (/home/runner/work/_actions/cycjimmy/semantic-release-action/v2/node_modules/semantic-release/index.js:170:22)
        at async release (/home/runner/work/_actions/cycjimmy/semantic-release-action/v2/src/index.js:19:18)
##[error]Node run failed with exit code 1
Complete job
```

**Workflow**
```yml
name: Release
on:
  push:
    branches:
      - master

jobs:
  test:
    name: Test
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v1
      - name: Setup Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 12
      - name: Install dependencies
        run: npm ci
      - name: Install Coverage Reporter
        run: |
          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
          chmod +x ./cc-test-reporter
          ./cc-test-reporter before-build
      - name: Run Tests
        run: npm test

  release:
    name: Release
    needs: Test
    runs-on: ubuntu-latest
    steps:
      - name: Semantic Release
        uses: cycjimmy/semantic-release-action@v2
        id: semantic   # Need an `id` for output variables
        with:
          branch: master
          # You can specify specifying version range for the extra plugins if you prefer.
          extra_plugins: |
            @semantic-release/git
            @semantic-release/changelog
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}

  upload-coverage:
    name: Upload Coverage
    runs-on: ubuntu-latest
    needs: Release
    steps:
      - name: Checkout
        uses: actions/checkout@v1
      - name: Setup Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 12
      - run: ./cc-test-reporter format-coverage -t lcov -o coverage/coverage.json coverage/lcov.info;
      - run: ./cc-test-reporter upload-coverage -i coverage/coverage.json;

  docs-build:
    needs: Release
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v1
      - name: Setup Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 12
      - name: Install dependencies
        run: npm ci
      - name: Build docs
        run: npm run storybook:build

  docs-deploy:
    runs-on: ubuntu-latest
    needs: docs-build
    steps:
      - name: Checkout
        uses: actions/checkout@v1
      - name: Setup Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 12
      - name: Build and Deploy
        uses: JamesIves/github-pages-deploy-action@releases/v3
        with:
          ACCESS_TOKEN: ${{ secrets.GH_TOKEN }}
          BRANCH: gh-pages # The branch the action should deploy to.
          FOLDER: _site # The folder the action should deploy.

```

**Expected behavior**
Release runs

--- a/README.md
+++ b/README.md
@@ -31,6 +31,8 @@ GitHub Action for [Semantic Release](https://github.com/semantic-release/semanti
 A simple example
 ```yaml
 steps:
+  - name: Checkout
+    uses: actions/checkout@v1
   - name: Semantic Release
     uses: cycjimmy/semantic-release-action@v2
     env:
@@ -41,6 +43,8 @@ steps:
 An advanced example
 ```yaml
 steps:
+  - name: Checkout
+    uses: actions/checkout@v1
   - name: Semantic Release
     uses: cycjimmy/semantic-release-action@v2
     id: semantic   # Need an `id` for output variables

